.travis.yml

```diff
@@ -40,7 +40,10 @@ matrix:
         - pycodestyle scipy benchmarks/benchmarks
         - |
           if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
-             python tools/lint_diff.py --branch $TRAVIS_BRANCH
+             git fetch --unshallow
+             git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
```

Comment:
Not sure if this works, but `origin/maintenance/1.*` is worth a try here. This step takes ~17 sec (rather than ~0 sec) to download a lot of data that's mostly not needed.

Reply:
It seems that [partial globbing is not supported](https://stackoverflow.com/a/15508750/2942522), so I just used `maintenance/*`; not sure that helps much but I did it.